examples/plugins/access_requests/requests.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+
+import logging
+from typing import Optional
+
+import pluggy
+
+from api.models import AccessRequest, OktaGroup, OktaUser
+from api.plugins import ConditionalAccessResponse
+
+request_hook_impl = pluggy.HookimplMarker('access_requests')
+logger = logging.getLogger(__name__)
+
+@request_hook_impl
+def access_request_created(access_request: AccessRequest,
+                           group: OktaGroup,
+                           requester: OktaUser) -> Optional[ConditionalAccessResponse]:
+    """Auto-approve memberships to the Auto-Approved-Group group"""
+
+    if not access_request.request_ownership and group.name == "Auto-Approved-Group":
```

Comment:
Will the groups have to be called 'Auto-Approved-Group' if we want to allow auto approval..? Maybe I'm missing how this fits in with the rest of the code...

Reply:
This is just a toy example of auto-approving membership to a single group. We can change the plugin code to approve groups by their name or some other criteria.